building/trusted_applications.rst

```diff
@@ -338,6 +338,68 @@ sequence:
         }
     }
 
+
+Identifying TA's client
+***********************
+
+GP TEE specification is mades so that TEE sessions, once established, are
```

Comment:
s/mades/made/
But, an alternative could be:
"The GP TEE specification is designed to ensure that TEE sessions are reliable once created."